src/hrflow_connectors/connectors/sapsuccessfactors/connector.py

```diff
@@ -0,0 +1,299 @@
+import datetime
+import re
+import typing as t
+from typing import Any, Dict
+
+from hrflow_connectors.connectors.hrflow.schemas import HrFlowProfile
+from hrflow_connectors.connectors.hrflow.warehouse import (
+    HrFlowJobWarehouse,
+    HrFlowProfileWarehouse,
+)
+from hrflow_connectors.connectors.sapsuccessfactors.warehouse import (
+    SAPJobWarehouse,
+    SAPProfileWarehouse,
+)
+from hrflow_connectors.core import (
+    BaseActionParameters,
+    Connector,
+    ConnectorAction,
+    WorkflowType,
+)
+
+from .schemas import SAPSuccessFactorsJob
+from .utils.datetime_converter import from_str_to_datetime
+
+DEFAULT_COUNTRY = "France"
+DEFAULT_JOB_NAME = "Undefined"
+DEFAULT_ADDRESS = "Undefined"
+
+
+def remove_html_tags(text: str) -> str:
+    """
+    Remove all HTML tags in a string
+    Args:
+        text (str): text to clean
+    Returns:
+        str: cleaned text (without HTML tags)
+    """
+    return re.sub("<[^<]+?>", "", text)
+
+
+def format_date(date: datetime.datetime) -> str:
+    """
+    Format `datetime.datetime` to SAP Date
+    Args:
+        date (datetime.datetime): Datetime to format
+    Returns:
+        str: formatted date
+    """
+    timestamp = int(date.timestamp())
+    sap_date = f"/Date({timestamp})/"
+    return sap_date
+
+
+def format_start_date(date_str: str) -> str:
+    """
+    Format ISO start date to SAP Date
+    Args:
+        date_str (str): iso date to format
+    Returns:
+        str: formatted SAP date
+    """
+    converted_date = from_str_to_datetime(date_str)
+    return format_date(converted_date)
+
+
+def format_end_date(date_str: str) -> str:
+    """
+    Format ISO end date to SAP Date
+    Args:
+        date_str (str): iso date to format
+    Returns:
+        str: formatted SAP date
+    """
+    converted_date = from_str_to_datetime(date_str)
+    # add 10 seconds to avoid that end_date is less or equal than start_date
+    converted_date += datetime.timedelta(seconds=10)
+    return format_date(converted_date)
+
+
+def format_job(data: SAPSuccessFactorsJob) -> t.Dict:
+    """
+    Format retrieved jobs into a HrFlow job object
+    Args:
+        data (SAPSuccessFactorsJob): job to format
+    Returns:
+        HrflowJob: job in the HrFlow job object format
+    """
+
+    job = dict()
+
+    jobRequisition = data.get("requisition")
+    if jobRequisition is None:
+        jobRequisition = dict()
```

Comment:
Tu peux modifié ta variable  jobRequisition en snake_case afin d'avoir de l'homogénéité dans le code